sqlglot/dialects/tsql.py

```diff
@@ -158,6 +158,9 @@ class Tokenizer(Tokenizer):
             "SQL_VARIANT": TokenType.VARIANT,
             "NVARCHAR(MAX)": TokenType.TEXT,
             "VARCHAR(MAX)": TokenType.TEXT,
+            "APPLY": TokenType.LATERAL,
+            "OUTER": TokenType.OUTER,
+            "LEFT OUTER": TokenType.LEFT,
```

Comment:
outer and left outer are already tokens, please remove them

Reply:
Ah yes, removed the dialect specific tokens